meilisearch/client.py

```diff
@@ -52,6 +52,32 @@ def create_index(self, uid, options=None):
     def get_indexes(self):
         """Get all indexes.
 
+        Returns
+        -------
+        indexes: list
+            List of Index instances.
+
+        Raises
+        ------
+        MeiliSearchApiError
+            An error containing details about why MeiliSearch can't process your request. MeiliSearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
+        """
+        response = self.http.get(self.config.paths.index)
+
+        return [
+            Index(
+                self.config,
+                x["uid"],
+                x["primaryKey"],
+                x["createdAt"],
+                x["updatedAt"],
+            )
+            for x in response
+        ]
```

Comment:
```suggestion
        return [
            Index(
                self.config,
                index["uid"],
                index["primaryKey"],
                index["createdAt"],
                index["updatedAt"],
            )
            for index in response
        ]
```

Just for readability, i hope its not bad practice

Reply:
`[x for x in ...]` is a pattern you see all the time in Python, but there is nothing saying it has to be that way. `index` is a perfectly acceptable alternative for `x` so I'll make this update.